src/lib/scale/evaluator.js

```diff
@@ -63,11 +63,17 @@ function extentsWrapper(useWholeData, clamp, pointsPerPxThreshold, minPointsPerP
 					clampedDomain[1]
 				];
 			}
-
+			const checkDirection = () => {
+				if (right < xAccessor(last(data))) {
+					return Math.min(right, xAccessor(last(data)));
+				} else {
+					return Math.min(right, xAccessor(last(data))) + (Math.min(right, xAccessor(last(data))) - clampedDomain[0]) * 0.2;
```

Comment:
In project, gap is equal to 0.15. If you set 0.2 here, then the gap in the project should also be 0.2. I think they should be the same.